components/clp-package-utils/clp_package_utils/controller.py

```diff
@@ -528,6 +530,34 @@ def _set_up_env_for_webui(self, container_clp_config: CLPConfig) -> EnvVarsDict:
 
         return env_vars
 
+    def _set_up_env_for_mcp_server(self) -> EnvVarsDict:
+        """
+        Sets up environment variables and directories for the MCP server component.
+
+        :return: Dictionary of environment variables necessary to launch the component.
+        """
+        component_name = MCP_SERVER_COMPONENT_NAME
+        logger.info(f"Setting up environment for {component_name}...")
```

Comment:
let's refer to the garbage collector for skipping the component creation

```suggestion
        if self._clp_config.mcp_server is None:
            logger.info(f"The MCP Server is not configured, skipping {component_name} creation...")
            return EnvVarsDict()
        logger.info(f"Setting up environment for {component_name}...")
```